apps/server/src/modules/deletion/services/references.service.ts

```diff
@@ -0,0 +1,27 @@
+import fs from 'fs';
+
+export class ReferencesService {
+	static loadFromTxtFile(filePath: string): string[] {
+		let fileContent = fs.readFileSync(filePath).toString();
+
+		// Replace all the CRLF occurrences with just a LF.
+		fileContent = fileContent.replace(/\r\n/g, '\n');
```

Comment:
Are you sure that you have considered all use cases?

Reply:
Anything specific in mind? Just a `\r` is no longer used anymore as far as I know, but if you think it should be added I can do it no problem. Unless you were thinking about something else?

Reply:
I only remembered something about the old project. Because of Windows/Linux/Mac. But I can no longer remember

Reply:
I've searched the Internet and I think the only case missing is if just the CR (`\r`) chars are used as a line separators. It was like that only in the old Macs, but to be sure I've also added support for this case: https://github.com/hpi-schul-cloud/schulcloud-server/pull/4521/commits/abc83e57d3fdc80d4d26c74c07bc7845fc628151 . Please let me know if you see any other cases.

Reply:
Confirmed offline with @SevenWaysDP there are no other known cases to cover.